### ISSUE TITLE
Errors parsing template comments

### ISSUE THREAD

**Issue by jbalsas** [metal/soyparser#1]
We currently ran into the following `soy` file:

```soy
{namespace AssetsInput}

/**
 * AssetsInput
 * @reference osb-faro-site-settings-web/components/settingsinput
 *
 * @param name
 * @param type
 * @param? disabled
 * @param? label
 * @param? portletNamespace
 * @param? value
 */
{template .render}
	<div class="form-group input-component">
		{if isNonnull($label)}
			<label for="{$name}">{$label}</label>
		{/if}

		<input
			data-onInput="onInput_"
			{if $disabled}disabled="disabled"{/if}
			id="{$portletNamespace ? $portletNamespace + $name : $name}"
			name="{$portletNamespace ? $portletNamespace + $name : $name}"
			type="{$type}"
			value="{$value}"
		/>
	</div>
{/template}
```

Which will fail when parsing the `@reference` comment, saying:

```
Parsing failed at 5:4. Expecting:
' '
'@param'
a character matching function (ch) {
      return str.indexOf(ch) < 0;
    }
```

We've removed that, since it was simply a token for something else, but I guess those should not get parsed, or maybe they should in a general way?

**Comment by mthadley** [comment]
@jbalsas I think ideally it should have been just parsed as comment content, like how `AssetsInput` would be. I see the problem though, I should be able to send a fix soon.

**Comment by mthadley** [comment]
@jbalsas Published `1.0.2`. Let me know how if you still have any issues.

**Comment by jbalsas** [comment]
Thanks a lot, @mthadley! 

I think we simply removed the weird tokens, but I'll keep an eye open in case we spot something else ;)
